src/packages/utils/src/things/json-rpc/json-rpc-base-types.ts

```diff
@@ -38,25 +47,26 @@ export class BaseJsonRpcType {
   }
 
   toString(): string | null {
-    if (this.bufferValue == null) {
+    const buffer = this._lazyBufferValue.getValue();
+    if (buffer == null) {
       return null;
     }
-    return `0x${this.bufferValue.toString("hex")}`;
+    return `0x${buffer.toString("hex")}`;
   }
 
   toBuffer(): Buffer {
-    return this.bufferValue;
+    return this._lazyBufferValue.getValue();
   }
 
   valueOf(): any {
-    return this.bufferValue;
+    return this._lazyBufferValue.getValue();
```

Comment:
Any reasons not to use this here:

```suggestion
    return this.toBuffer();
```